Formula/aliyun-cli.rb

```diff
@@ -0,0 +1,29 @@
+class AliyunCli < Formula
+  desc "Universal Command-line Interface for Alibaba Cloud"
+  homepage "https://github.com/aliyun/aliyun-cli"
+  url "https://github.com/aliyun/aliyun-cli/archive/v3.0.5.tar.gz"
+  sha256 "fa9a9e736f3a02c454695b75e0dd1e7c140fba63d114537d24babd0b6930f99a"
+
+  head do
+    url "https://github.com/aliyun/aliyun-cli.git"
+  end
+
+  depends_on "go" => :build
+  depends_on "go-bindata" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    dir = buildpath/"src/github.com/aliyun/aliyun-cli"
```

Comment:
How are dependencies/submodules handled? Are they included in the tarball, or downloaded? Are versions pinned?

Reply:
They are vendored in the project